packages/components/Avatar/src/useAvatar.ts

```diff
@@ -43,6 +46,19 @@ export const useAvatar = (props: AvatarProps): AvatarInfo => {
     showBadge,
   };
 
+  let imageProps: ImageProps = {
+    accessibilityLabel,
+    source: imageUrl ? ({ uri: imageUrl } as ImageSourcePropType) : undefined,
+  };
+
+  if (image && image.source) {
+    const { source, ...restImageProps } = image;
+    imageProps = {
```

Comment:
Do we want accessibilityLabel to stay or do you think the imageProps prop should take care of that? Asking since this code now reassigns imageProps so that accessibilityLabel that was set on line 50 isn't part of it anymore.

Reply:
I think it would be good to add a check there that if there's no accessibilityLabel in image object then to use accessibilityLabel from the Avatar